python/cugraph/cugraph/dask/centrality/betweenness_centrality.py

```diff
@@ -184,6 +215,164 @@ def betweenness_centrality(
+
+    else:
+        if k is not None:
+            k = get_distributed_data(k)
+            wait(k)
+            k = k.worker_to_parts
+
+    if input_graph.renumbered:
+        if isinstance(k, dask_cudf.DataFrame):
+            tmp_col_names = k.columns
+
+        elif isinstance(k, dask_cudf.Series):
+            tmp_col_names = None
+
+        if isinstance(k, (dask_cudf.DataFrame, dask_cudf.Series)):
+            k = input_graph.lookup_internal_vertex_id(k, tmp_col_names)
+
+    # FIXME: should we add this parameter as an option?
+    do_expensive_check = False
+
+    client = get_client()
+
+    ddf = _mg_call_plc_betweenness_centrality(
+        input_graph=input_graph,
+        client=client,
+        sID=Comms.get_session_id(),
+        k=k,
+        random_state=random_state,
+        normalized=normalized,
+        endpoints=endpoints,
+        do_expensive_check=do_expensive_check,
+    )
+
+    if input_graph.renumbered:
+        return input_graph.unrenumber(ddf, "vertex")
+
+    return ddf
+
+
+def edge_betweenness_centrality(
+    input_graph,
+    k: Union[
+        int, list, cudf.Series, cudf.DataFrame, dask_cudf.Series, dask_cudf.DataFrame
+    ] = None,
+    normalized: bool = True,
+    weight: cudf.DataFrame = None,
+    random_state: int = None,
+) -> dask_cudf.DataFrame:
+    """
+    Compute the edge betweenness centrality for all edges of the graph G.
+    Betweenness centrality is a measure of the number of shortest paths
+    that pass over an edge.  An edge with a high betweenness centrality
+    score has more paths passing over it and is therefore believed to be
+    more important.
+
+    To improve performance. rather than doing an all-pair shortest path,
+    a sample of k starting vertices can be used.
+
+    CuGraph does not currently support 'weight' parameters as seen in the
+    corresponding networkX call.
+
+    Parameters
+    ----------
+    input_graph: cuGraph.Graph
+        The graph can be either directed (Graph(directed=True)) or undirected.
+        Weights in the graph are ignored, the current implementation uses a parallel
+        variation of the Brandes Algorithm (2001) to compute exact or approximate
+        betweenness. If weights are provided in the edgelist, they will not be
+        used.
+
+    k : int, list or (dask)cudf object or None, optional (default=None)
+        If k is not None, use k node samples to estimate betweenness.  Higher
+        values give better approximation.  If k is either a list or a (dask)cudf,
+        use its content for estimation: it contain vertex identifiers. If k is None
+        (the default), all the vertices are used to estimate betweenness.  Vertices
+        obtained through sampling or defined as a list will be used as sources for
+        traversals inside the algorithm.
+
+    normalized : bool, optional (default=True)
+        If True normalize the resulting betweenness centrality values
```

Comment:
This is described elsewhere as:
"Normalization will ensure that values are in [0, 1]."
Should that also be done here?